src/api/registration/EditorRegistration.ts

```diff
@@ -41,8 +45,16 @@ export interface EditorRegistration<PARAMS = EmptyParameters> extends Partial<Ed
     /**
      * Should this be a dry run editor: That is,
      * should it wait for the build result to determine whether to raise a pull request
-     * or issue. Default is no.
+     * or issue. Default is editorCommand, which creates a branch and a PR.
      */
-    dryRun?: boolean;
+    editorCommandFactory?: EditorCommandFactory<PARAMS>;
```

Comment:
Not sure the comment is quite right now. Isn't this a more general mechanism?

Reply:
yes good point